tests/test_virtualenv.py

```diff
@@ -104,10 +104,10 @@ def test_exists(self, exists, mock_local):
             mock.call("/path/to/python.py -m virtualenv '/my/local path/to/venv'")
         ]
 
-    @pytest.mark.parametrize('upgrade_pip', [True, False])
+    @pytest.mark.parametrize('upgrade_pip', [True, False], 'upgrade_wheel', False)
```

Comment:
I'm thinking this line is the cause of the erroring tests.
http://doc.pytest.org/en/latest/parametrize.html#_pytest.python.Metafunc.parametrize